Makefile

```diff
@@ -3,6 +3,8 @@ SCRIPTS_INSTALL_PATH = /lib/modules/lua
 LUNATIK_INSTALL_PATH = /usr/local/sbin
 LUA_API = lua/lua.h lua/lauxlib.h lua/lualib.h
 KDIR ?= ${MODULES_INSTALL_PATH}/build
+RM = rm -f
+MKDIR = mkdir -p
```

Comment:
Please, split this in another commit or update the commit message to reflect all your changes..

Also, I think you could add the permission to this var as well..